lib/providers/notifiers/connection_notifier.dart

```diff
@@ -13,31 +19,48 @@ class ConnectionNotifier extends ChangeNotifier {
     'https://mainnet.telosusa.io',
     'https://telos.eosphere.io',
     'https://telos.caleos.io',
+    'https://api.eos.miami',
+    'https://hyperion.telosgermany.io',
   ];
 
   void init() {
     discoverEndpoints();
   }
 
   void discoverEndpoints() async {
+    List<Future> checks = List<Future>();
+
     for (var endpoint in availableEndpoints) {
-      var ping = Stopwatch()..start();
+      checks.add(checkEndpoint(endpoint));
+    }
 
-      Response res = await get("$endpoint/v2/health");
+    var responses = await Future.wait(checks);
 
-      ping.stop();
+    responses.sort((a, b) => a.ping - b.ping);
 
-      if (res.statusCode == 200) {
-        int endpointPing = ping.elapsedMilliseconds;
+    //for (int i=0; i<responses.length; i++) {
```

Comment:
dead code?

Reply:
i do like leaving my testing code in there in case it breaks again, but ... right, removed.